### PR TITLE
Remove popcorn-time plugin

### PR DESCRIPTION
**Pull request by DStewart1997**
Popcorn time is no longer up and running so fails to install.

### CODE REVIEW

**Comment by denisotugo** [comment]
It is still up http://popcorn-time.se/

File: plugins/popcorntime.plugin/install.sh [hunk 1]
```diff
@@ -1,42 +0,0 @@
...
-if [[ "$(uname -m)" = "x86_64" ]]; then
-	ARCH="64"
-else
-	ARCH="32"
-fi
-
-mkdir -p "$CACHEDIR"
-cd "$CACHEDIR"
-
-URL=$(wget "https://popcorntime.io" -O - | grep -o "https://get.popcorntime.io/build/\S*${ARCH}.tar.xz" | head -n 1)
```
Comment (denisotugo) [inline plugins/popcorntime.plugin/install.sh]: http://popcorn-time.se/Popcorn-Time-linux32.tar.gz / http://popcorn-time.se/Popcorn-Time-linux64.tar.gz

Comment (denisotugo) [inline plugins/popcorntime.plugin/install.sh]: they changed website

**Comment by DStewart1997** [comment]
No, that isn't the same one that was used before.

There are two main versions of popcorn time, IO (The one Fedy was using before, that is [now shutdown](http://arstechnica.com/tech-policy/2015/11/mpaa-shuts-down-major-torrent-sites-including-popcorn-time/)) and SE (The one you mention, formerly known as Time4Popcorn).

SE has been known to infect users with malware but it is pretty debated.

### Against SE
[One](https://www.reddit.com/r/PopCornTime/comments/3pzm7c/any_reason_not_to_use_popcorntimese/cwaw1pe)
[Two](https://www.reddit.com/r/PopCornTime/comments/3q40c3/was_there_ever_any_proof_that_the_se_version_had/cwc01b2)
[Three](https://www.reddit.com/r/PopCornTime/comments/3q40c3/was_there_ever_any_proof_that_the_se_version_had/cwbxeg6)
[Four](https://www.reddit.com/r/PopCornTime/comments/2lyxnm/time4popcorn_popcorntimese_now_includes_adware_do/)

### For SE
[One](https://www.reddit.com/r/PopCornTime/comments/3pzm7c/any_reason_not_to_use_popcorntimese/cwau7xp)
[Two](https://www.reddit.com/r/time4popcorn/comments/24a7qd/why_time4popcorn_is_incredibly_dangerous/ch7djv6)
[Three - Comment from SE devs](https://www.reddit.com/r/time4popcorn/comments/24a7qd/why_time4popcorn_is_incredibly_dangerous/ch7wenf)

It seems well accepted that SE's source code is always out of date (by at least a few months) - the main reason so much debate over the issues happens.

Their (github repo)[https://github.com/time4popcorn/popcorntime-desktop] seems to be shutdown and I cannot access their [website](popcorn-time.se/) as it is blocked by my ISP.

I'd leave it to @Abhinav1217 to decide if it should be added to Fedy.


The original popcorn time io can still be built from [source](https://github.com/popcorn-official/popcorn-desktop) and there are workarounds to make the [TV shows](https://www.reddit.com/r/PopCornTime/comments/3rozk6/working_tv_api_hidden_service/) and [movies](https://www.reddit.com/r/PopCornTime/comments/3rk6st/movie_section_workaround/) work again.


The devs of IO have now moved to a fully legal alternative called the [Butter Project](http://butterproject.org/) which will only contain non copyright media - but they are planning to implement a plugin system - it is still early in development but can be built from [source](https://github.com/butterproject/butter-desktop) for Linux and MacOS (Although there isn't much to see yet).

**Comment by Abhinav1217** [comment]
I will remove popcorn time for now. Until a better Alternative is suggested by community.  I would wait for butter project to mature a bit more.  Maybe I will create a popcorn time plugin which will build it from source.

**Comment by Abhinav1217** [comment]
Any body used http://strem.io/ .?  It is not open source though, I think.

**Comment by DStewart1997** [comment]
@Abhinav1217 I was also thinking of a popcorn time plugin to build from source but thought it would only really be worthwhile if we can also apply the workarounds.

I haven't used stremio yet, skeptical due to it being closed source but it seems pretty popular - I've read they're also planning on becoming open souce but can't find any mention on their site.

**Comment by Abhinav1217** [comment]
I already removed it.

**Pull request closed without merging**